lib/web/apiserver.go

```diff
@@ -91,6 +92,14 @@ import (
 const (
 	// SSOLoginFailureMessage is a generic error message to avoid disclosing sensitive SSO failure messages.
 	SSOLoginFailureMessage = "Failed to login. Please check Teleport's log for more details."
+
```

Comment:
Latest discussion about the rate limit: https://gravitational.slack.com/archives/C0509EYASCW/p1683738890958669